tests/resiliparse/test_process_guard.py

```diff
@@ -100,7 +100,7 @@ def wait_func_signal_term_escalate():
                 pass
 
 
-@time_guard(timeout_ms=250, grace_period_ms=0, check_interval=80, interrupt_type=InterruptType.exception)
+@time_guard(timeout_ms=2500, grace_period_ms=0, check_interval=80, interrupt_type=InterruptType.exception)
```

Comment:
I had to add this here since this (and only this test) took quite long in a QEMU emulated environment and failed because of the timeout. Did not spend much time at all for investigating why this takes so long.